sys/net/gnrc/netif2/gnrc_netif2.c

```diff
@@ -1138,6 +1139,7 @@ static void *_gnrc_netif2_thread(void *args)
 #endif
     if (netif->ops->init) {
         netif->ops->init(netif);
+        puts("inti device");
```

Comment:
What?

Reply:
Sorry, this is just for debug, forget to remove it.. :-P Will remove!